frontend/package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Elm client example for our mu-haskell GraphQL server!",
   "scripts": {
     "codegen": "elm-graphql --schema-file ../library.graphql --base LibraryApi --output src",
-    "start": "elm-app start"
+    "start": "elm-live src/Main.elm --dir=public --open --host=0.0.0.0 --port=8080 --pushstate --start-page=index.html -- --output=public/bundle.js"
```

Comment:
Even though the dependencies are much less heavy and the package is better maintained... this line here is terrible DX in my opinion 🤣  Why not make all those arguments the default!? 🤔